src/commands/QueryAdminDetails.tsx

```diff
@@ -0,0 +1,123 @@
+
+export type MatrixHomeserver = string;
+
+makePresentationType({
+    name: "MatrixHomeserver",
+    // This is a very very crude way to detect a url.
+    validator: simpleTypeValidator("MatrixHomeserver", (readItem: ReadItem) => (readItem instanceof String) && (!readItem.includes('#') || !readItem.includes('!')))
+})
+
+interface SupportJson {
+    contacts?: {
+        matrix_id?: string,
+        email_address?: string;
+        role: "admin" | "security";
+    }[],
+    support_page?: string;
+}
+
+async function queryAdminDetails(
+    this: MjolnirContext,
+    _keywords: ParsedKeywords,
+    entity: UserID | MatrixHomeserver | string
+): Promise<CommandResult<[UserID | MatrixHomeserver | string, SupportJson], CommandError>> {
+    let domain: string;
+    if (entity instanceof UserID) {
+        domain = `https://${entity.domain}`;
+    } else {
+        // Doing some cleanup on the url
+        if (!entity.startsWith("https://") && !entity.startsWith("http://")) {
+            domain = `https://${entity}`;
+        } else {
+            domain = entity;
+        }
+    }
+
+
+    try {
+        const resp: SupportJson = await new Promise((resolve, reject) => {
+            getRequestFn()(`${domain}/.well-known/matrix/support`, (error: any, response: any, resBody: unknown) => {
+                if (error) {
+                    reject(new CommandError(`The request failed with an error: ${error}.`));
+                } else if (response.statusCode !== 200) {
+                    reject(new CommandError(`The server didn't reply with a valid response code: ${response.statusCode}.`));
+                } else if (typeof resBody === 'object' && resBody !== null && ('contacts' in resBody || 'support_page' in resBody)) {
+                    resolve(resBody as SupportJson)
+                } else if (resBody === null) {
+                    reject(new CommandError(`The response was empty.`));
+                } else {
+                    reject(new CommandError(`Don't know what to do with response body ${resBody}. Assuming its not a json`));
+                }
+            });
+        });
+        return CommandResult.Ok([entity, resp]);
+    } catch (error: any) {
+        return CommandResult.Err(error);
+    }
+}
+
+defineInterfaceCommand({
+    designator: ["queryAdmin"],
+    table: "mjolnir",
+    parameters: parameters([
+        {
+            name: "entity",
+            acceptor: union(
+                findPresentationType("UserID"),
+                findPresentationType("MatrixHomeserver"),
+                findPresentationType("string")
+            )
+        }
+    ]),
+    command: queryAdminDetails,
+    summary: "Queries the admin of the Homeserver or user using MSC1929 if available",
+})
+
+function renderSupportJson([entity, support_json]: [UserID | MatrixHomeserver | string, SupportJson],): DocumentNode {
+    if (!support_json.support_page) {
+        return <root>
+            <b>Support infos for ({entity}):</b><br />
```

Comment:
I think info is plural right? 

![image](https://github.com/the-draupnir-project/Draupnir/assets/50846879/fbab3f36-ba2f-4829-bd75-3df8d0666499)

Reply:
Oops. German problem. We have a dedicated plural 😅

Reply:
Fixed in 506ae54